CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ rewritten to their query representation
 - Add `ABS` function support on `INTERVAL` field
 - Display of `Ion` in PartiQL CLI output
 - Support `INTERVAL` times and divide with numerics
+- `missing` propagation behavior in functions args
```

Comment:
nit: let's be a bit more specific in this description to mention it's for when `missing` is included with `null` in the function args